tests/blockchain/test_blockchain.py

```diff
@@ -6,6 +6,7 @@
 from dataclasses import replace
 from secrets import token_bytes
 from typing import Optional
+from conftest import db_version
```

Comment:
No need to import.  Fixtures exist in the magical backworld of pytest.  And yes, I use 'magical' in a negative way.